docs/tabpy-tools.md

```diff
@@ -136,6 +137,95 @@ client.remove('WillItDefault')
 
 ```
 
+## Predeployed Functions
```

Comment:
@nmannheimer Are you proofreading these documentation updates?

Reply:
@nmannheimer can you please review the docstrings I added to PCA and SentimentAnalysis as well

Reply:
Yep! Reviewing now.